app/views/followups/_followup.html.erb

```diff
@@ -20,13 +20,13 @@
                          name_for_followup: name_for_followup } %>
   <% end %>
 
-  <% if @info_request.who_can_followup_to(incoming_message).count > 0 %>
+  <% if incoming_message && @info_request.who_can_followup_to(incoming_message).count > 0 %>
     <div id="other_recipients" class="box other_recipients">
       <%= _("Don't want to address your message to {{person_or_body}}?  You can also write to:", :person_or_body => name_for_followup) %>
 
       <ul>
         <% @info_request.who_can_followup_to(incoming_message).each do |name, email, id|  %>
-          <% if id.nil? && !incoming_message.nil? && incoming_message.valid_to_reply_to? %>
+          <% if id.nil? && incoming_message.valid_to_reply_to? %>
```

Comment:
of course in doing this, I suddenly realise that we are checking the `incoming_message` is valid to reply to in order to show the main contact address, which makes my head hurt